lib/zinnia.js

```diff
@@ -53,6 +53,9 @@ const maybeReportErrorToSentry = (/** @type {unknown} */ err) => {
   /** @type {Parameters<Sentry.captureException>[1]} */
   const hint = { extra: {} }
   if (typeof err === 'object') {
+    if ('shouldReportToSentry' in err && err.shouldReportToSentry === false) {
```

Comment:
An alternative name to consider: `reportedToSentry` or `wasReportedToSentry`.

Instead of setting this flag in various places where we want to report errors, maybe we can set the flag here, inside `maybeReportErrorToSentry`? That will prevent double-reports everywhere, not only in the place you modify in this PR.

```js
const maybeReportErrorToSentry = (/** @type {unknown} */ err) => {
  // ...
  if ('reportedToSentry' in err && err.reportedToSentry) return
  err.reportedToSentry = true
  // ...
}
```

Reply:
<img width="567" alt="Screenshot 2024-04-29 at 16 15 10" src="https://github.com/filecoin-station/core/assets/10247/2e4c4afd-dfe5-43d5-b02f-a15e26eca3a4">

Do you know how to fix this?